app/gui2/src/stores/graph/index.ts

```diff
@@ -262,24 +262,11 @@ export const useGraphStore = defineStore('graph', () => {
 
   function normalizeVisMetadata(
     id: Opt<VisualizationIdentifier>,
-    visible?: boolean,
```

Comment:
Why this change? I always thought this is identical?

Reply:
this is (hopefully?) to ensure that users cannot omit `visible`, so we don't accidentally lose `visible` by forgettting to pass it

Reply:
You're right, TS complains about missing argument.